docs/public/RELEASE.md

```diff
@@ -0,0 +1,49 @@
+# Release v2024.1 (2024-07-01)
+
+### [Release information]
+ - Translations have not been fully updated in this release. They will be updated in an upcoming release.
+
+### [Breaking changes]
```

Comment:
https://github.com/zonemaster/zonemaster-engine/pull/1339 is also breaking.

Reply:
In what way is it breaking?

Reply:
The profile.json file in the dist directory used to be part of the public interface of Zonemaster Engine but it isn't anymore.

Reply:
But it is still there. Nothing is breaking.

Reply:
I agree that there is a file there but its contents does not reflect the default file. If you want to extract the default profile you need to use the command.

Reply:
Included.